Formula/mujs.rb

```diff
@@ -6,6 +6,7 @@ class Mujs < Formula
       tag:      "1.3.3",
       revision: "57e3f01d5f29c5823be725d96284488edf5f8ae1"
   license "ISC"
+  revision 1
```

Comment:
```suggestion
```
Actually, I don't think we really need this either. It doesn't look like most users really care about the broken `.pc` file -- we can always bump the `revision` later if there are more complaints.

Reply:
I care, also, without a revision here, the solution for this issue is reinstall from source, which is not enabled by default any more.
I rather to fix it now than wait for another mujs, which may happen a fews months later

Reply:
You don't need to reinstall from source if there is no `revision`. You just need to do `brew reinstall mujs`.